.gitignore

```diff
@@ -5,4 +5,5 @@ compile_commands.json
 .ycm_extra_conf.py
 cscope.*
 third_party/glslang/
+third_party/googletest/
 third_party/gmock-1.7.0/
```

Comment:
I intentionally left it in for the transition period where people will have both trees.  But I can see it's misleading.  Will remove it.